### PR TITLE
Chore: anonymous classes replaced with lambda expressions

### PR DESCRIPTION
**Pull request by dbmalkovsky**
Part 3.  Other minor formatting, annotation changes, etc.

### CODE REVIEW

File: src/main/java/org/isf/medicalstockward/gui/WardPharmacy.java [hunk 1]
```diff
@@ -956,37 +952,35 @@ private JPanel getJPanelMedicalsSearch() {
...
-				} catch (OHServiceException e1) {
-					medicals = null;
-					OHServiceExceptionUtil.showMessages(e1);
-				}
-				MedicalType medicalType;
-				if (jComboBoxTypes.getSelectedItem() instanceof String) {
-					medicalType = null;
-				} else {
-					medicalType = (MedicalType) jComboBoxTypes.getSelectedItem();
+		searchButton.addActionListener(ae -> {
```
Comment (mwithi) [inline src/main/java/org/isf/medicalstockward/gui/WardPharmacy.java:955]: another case `ae` (just a note)

File: src/main/java/org/isf/medicalstockward/gui/WardPharmacyNew.java [hunk 2]
```diff
@@ -274,16 +271,12 @@ private JTextField getJTextFieldUse() {
...
-					jButtonTrashPatient.setEnabled(false);
-					jTextFieldUse.setEnabled(true);
-                                        wardBox.setEnabled(false);
-				}
+			jRadioUse.addActionListener(e -> {
+				jTextFieldPatient.setEnabled(false);
+				jButtonPickPatient.setEnabled(false);
+				jButtonTrashPatient.setEnabled(false);
+				jTextFieldUse.setEnabled(true);
+wardBox.setEnabled(false);
```
Comment (mwithi) [inline src/main/java/org/isf/medicalstockward/gui/WardPharmacyNew.java:279]: indentation?

File: src/main/java/org/isf/medicalstockward/gui/WardPharmacyNew.java [hunk 3]
```diff
@@ -770,19 +703,15 @@ private JRadioButton getJRadioPatient() {
...
-                                        wardBox.setEnabled(false);
-					if (patientSelected != null) {
-						jButtonTrashPatient.setEnabled(true);
-					}
-					
+			jRadioPatient.addActionListener(e -> {
+				jTextFieldUse.setEnabled(false);
+				jTextFieldPatient.setEnabled(true);
+				jButtonPickPatient.setEnabled(true);
+wardBox.setEnabled(false);
```
Comment (mwithi) [inline src/main/java/org/isf/medicalstockward/gui/WardPharmacyNew.java:710]: also here

**Review by mwithi — APPROVED** [review]
LGTM. Thanks!

File: src/main/java/org/isf/medicalstockward/gui/WardPharmacyNew.java (hunk 2 above)
Comment (dbmalkovsky) [reply src/main/java/org/isf/medicalstockward/gui/WardPharmacyNew.java:279]: fixed

File: src/main/java/org/isf/medicalstockward/gui/WardPharmacyNew.java (hunk 3 above)
Comment (dbmalkovsky) [reply src/main/java/org/isf/medicalstockward/gui/WardPharmacyNew.java:710]: fixed

File: src/main/java/org/isf/medicalstockward/gui/WardPharmacy.java (hunk 1 above)
Comment (dbmalkovsky) [reply src/main/java/org/isf/medicalstockward/gui/WardPharmacy.java:955]: ok

**Comment by mwithi** [comment]
It seems that part 3 has conflict over part 2

**Pull request merged**